src/po/Make_mvc.mak

```diff
@@ -78,6 +75,9 @@ checkclean:
 
 converted: $(MOCONVERTED)
 
+checklanguage:
+	@if "$(LANGUAGE)"=="" (echo Set the environment variable ^%LANGUAGE^%. See README_mvc.txt. && exit 1)
+
```

Comment:
it's a lot better than it was.

Reply:
Since we are checking the system environment variable, it seems to me that this code should be like this:
```
checklanguage:
	@if NOT DEFINED LANGUAGE (echo Set the environment variable ^%LANGUAGE^%. See README_mvc.txt. && exit 1)
```

Reply:
It doesn't work when `LANGUAGE` is set from the command line.
E.g. `nmake -f Make_mvc LANGUAGE=xy`

Reply:
When %LANGUAGE% is cleared:
```
E:\Project\vim\en-ru\po\tst\tst2>echo %LANGUAGE%
%LANGUAGE%

E:\Project\vim\en-ru\po\tst\tst2>E:\Project\vim\tools\nmake.exe -f Make_mvc.mak install

Microsoft (R) Program Maintenance Utility Version 14.27.29111.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        if NOT DEFINED LANGUAGE (echo Set the environment variable ^%LANGUAGE^%. See README_mvc.txt. && exit 1)
Set the environment variable %LANGUAGE%. See README_mvc.txt.
NMAKE : fatal error U1077: 'if' : return code '0x1'
Stop.
```
When setting the %LANGUAGE% variable from the command line:
```
E:\Project\vim\en-ru\po\tst\tst2>E:\Project\vim\tools\nmake.exe -f Make_mvc.mak LANGUAGE=ru install

Microsoft (R) Program Maintenance Utility Version 14.27.29111.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        if NOT DEFINED LANGUAGE (echo Set the environment variable ^%LANGUAGE^%. See README_mvc.txt. && exit 1)
        set OLD_PO_FILE_INPUT=yes
        "D:\Program Files\GetText\bin\msgfmt" -v -o ru.mo ru.po
2943 переведённых сообщения, 4 неточных перевода.
        if not exist ..\..\runtime\lang\ru\LC_MESSAGES mkdir ..\..\runtime\lang\ru\LC_MESSAGES
        copy /y ru.mo ..\..\runtime\lang\ru\LC_MESSAGES\vim.mo
Скопировано файлов:         1.
```
At least on my Windows NT 6.1 it works. Although I admit that it may not work on other systems or configurations. I'll have to check it out.